src/components/AddBook.js

```diff
@@ -18,14 +19,12 @@ const AddBook = (props) => {
   };
 
   const clickAdd = () => {
-    const { listenChanges } = props;
     if (title === '' || author === '') {
       console.log('form is empty');
```

Comment:
- [ ] For best practice, kindly remove all `conslole.log()` it makes your code slower